Common/Tests/Insights/HyperwalletInsightsTests.swift

```diff
@@ -75,38 +105,62 @@ class HyperwalletInsightsTests: XCTestCase {
         XCTAssertNotNil(HyperwalletInsights.shared, "HyperwalletInsights should be initialized")
         let params = [countryTag: country, currencyTag: currency]
         HyperwalletInsights.shared.trackImpression(pageName: pageName, pageGroup: pageGroup, params: params)
+        sleep(2)
```

Comment:
Why sleep instead of expectation?

Reply:
After the method `trackImpression` execution, property `insights` is a real instance, not a mock.